casper/src/main/scala/coop/rchain/casper/util/rholang/CasperRhoRuntimeSyntax.scala

```diff
@@ -220,12 +220,13 @@ final class RhoRuntimeOps[F[_]: Sync: Span: Log](
 
   def computeGenesis(
       terms: Seq[Signed[DeployData]],
-      blockTime: Long
+      blockTime: Long,
+      blockNumber: Long
   ): F[(StateHash, StateHash, Seq[ProcessedDeploy])] =
     Span[F].traceI("compute-genesis") {
       for {
         _ <- runtime.setBlockData(
-              BlockData(blockTime, 0, PublicKey(Array[Byte]()), 0)
+              BlockData(blockTime, blockNumber, PublicKey(Array[Byte]()), 0)
```

Comment:
The fix here is not actually the real problem. I just want to change it to be consistent.

Reply:
Good catch!